config/settings.js

```diff
@@ -27,11 +27,20 @@
  */
 
 window.spinnakerSettings = {
+  defaultProviders: ['aws', 'gce', 'azure', 'cf', 'kubernetes'],
+  feedbackUrl: process.env.FEEDBACK_URL,
   gateUrl: gateUrl,
-  bakeryDetailUrl: bakeryBaseUrl + '/api/v1/global/logs/{{context.status.id}}?html=true',
+  bakeryDetailUrl: bakeryBaseUrl + '/api/v1/global/logs/{{context.status.id}}?html=true'
+  authEndpoint: process.env.AUTH_ENDPOINT,
   pollSchedule: 30000,
-  defaultTimeZone: 'America/New_York', // see http://momentjs.com/timezone/docs/#/data-utilities/
+  defaultTimeZone: process.env.TIMEZONE || 'America/Los_Angeles', // see http://momentjs.com/timezone/docs/#/data-utilities/
```

Comment:
It would be nice to move this value into one of the `spinnaker.yml` files and leave it as-is in Deck's settings.js file itself. I will talk to @ewiseblatt today and see if that makes sense.